test/Spec.hs

```diff
@@ -1,10 +1,58 @@
+import Data.List
+import Data.Function
 import Boids
 import Test.HUnit
-import Test.HUnit.Approx
 import System.Exit
 
+testIsWithinView :: Test
+testIsWithinView  = TestCase (assertBool "Seeing a boid behind me" (not $ isWithinView boid1 boid2))
```

Comment:
As far as I can tell, this test is about *not* seeing the boid behind. Is description misleading, or is it a proper way to write tests in HUnit (I have no experience with this particular framework)?